frontend/src/components/landing/BundleBanner.tsx

```diff
@@ -0,0 +1,134 @@
+import { Localized, useLocalization } from "@fluent/react";
+import { getBundlePrice } from "../../functions/getPlan";
+import { RuntimeData } from "../../hooks/api/runtimeData";
+import { Button } from "../Button";
+import { MozillaVpnWordmark } from "../Icons";
+import styles from "./BundleBanner.module.scss";
+import womanInBanner from "./images/bundle-banner-woman.png";
+import bundleFloatOne from "./images/bundle-float-1.svg";
+import bundleFloatTwo from "./images/bundle-float-2.svg";
+import bundleFloatThree from "./images/bundle-float-3.svg";
+import bundleLogo from "./images/vpn-and-relay-logo.svg";
+
+type FloatingFeaturesProps = {
+  icon: string;
+  text: string;
+  position: string;
+};
+
+export type Props = {
+  runtimeData: RuntimeData;
+};
+
+const FloatingFeatures = (props: FloatingFeaturesProps) => {
+  return (
+    <div
+      className={`${styles[props.position]} ${styles["float-features-item"]}`}
+    >
+      <img alt={props.text} src={props.icon} />
+      <span className={styles["float-features-text"]}>{props.text}</span>
+    </div>
+  );
+};
+
+export const BundleBanner = (props: Props) => {
+  const { l10n } = useLocalization();
+
+  const mainImage = (
+    <img
+      src={womanInBanner.src}
+      alt="Woman in Banner"
```

Comment:
This `alt` isn't localised, though as it's just purely decorative, I think it's more helpful for a screen reader user to just not have it read out anything:

```suggestion
      alt=""
```